packages/composite-checkout/src/wpcom/wp-checkout-order-summary.js

```diff
@@ -23,8 +23,8 @@ export default function WPCheckoutOrderSummary() {
 		<React.Fragment>
 			<SummaryContent>
 				<ProductList>
-					{ items.map( product => {
-						return <ProductListItem key={ product.id }>{ product.label }</ProductListItem>;
+					{ items.map( ( product, index ) => {
```

Comment:
What is the `index` being used for here? The React docs [recommend against using it as a key](https://reactjs.org/docs/lists-and-keys.html#keys).